pynecone/pc.py

```diff
@@ -83,6 +83,9 @@ def run(
     frontend_port = get_config().port if port is None else port
     backend_port = get_config().backend_port if backend_port is None else backend_port
 
+    # set the upload url in pynecone.json file
+    build.set_pynecone_upload_endpoint()
```

Comment:
I like this idea - we currently compile many constants right into the pages file and pass them into functions in `state.js` (event the event url). I think we add more constants here. Maybe we create a new file though instead of using the `pynecone.json`

Reply:
created a follow up issue to handle this here #1012